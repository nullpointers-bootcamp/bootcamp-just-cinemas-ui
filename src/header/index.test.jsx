import React from "react";
import renderer from "react-test-renderer";
import Header from "./index";

it("show header", () => {
  const tree = renderer.create(<Header />).toJSON();
  expect(tree).toMatchSnapshot();
});
