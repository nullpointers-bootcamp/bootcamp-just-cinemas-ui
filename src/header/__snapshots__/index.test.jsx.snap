// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`show header 1`] = `
<a
  href="/"
  onClick={[Function]}
>
  <img
    alt="just cinemas"
    className="logo"
    src="https://drive.google.com/uc?export=view&id=1d3o9sOOiobgOEh7HWiCJBOHKTsmXopTX"
  />
</a>
`;
